include/onnxruntime/core/session/onnxruntime_lite_custom_op.h

```diff
@@ -243,15 +292,105 @@
   }
 
  private:
-  size_t indice_;
-  bool is_input_;
   std::vector<char> chars_;                           // for input
   std::vector<std::string_view> input_string_views_;  // for input
 };
 
 using TensorPtr = std::unique_ptr<Custom::TensorBase>;
+using TensorPtrs = std::vector<TensorPtr>;
+
+struct Variadic : public TensorBase {
```

Comment:
>Variadic [](http://example.com/codeflow?start=7&length=8)

I think "TensorList" or "TensorArrary" is a better name...

Reply:
Let's go with TensorArray.